cli/tests/it/cast.rs

```diff
@@ -355,3 +355,20 @@ casttest!(cast_receipt_revert_reason, |_: TestProject, mut cmd: TestCommand| {
     assert!(output.contains("revertReason"));
     assert!(output.contains("Transaction too old"));
 });
+
+casttest!(cast_access_list, |_: TestProject, mut cmd: TestCommand| {
+    let rpc = next_http_rpc_endpoint();
+    cmd.args([
+        "access-list",
+        "0xbb2b8038a1640196fbe3e38816f3e67cba72d940",
+        "skim(address)",
+        "0xbb2b8038a1640196fbe3e38816f3e67cba72d940",
+        "--rpc-url",
+        rpc.as_str(),
+        "--gas-limit", // need to set this for alchemy.io to avoid "intrinsic gas too low" error
```

Comment:
This is an Alchemy specific issue. Tested against a local `anvil` and `ankr` without needing to specify gas limit. 
The output should be stable. Simply calling `skim()` on the WBTC/WETH UniswapV2 pair, which is immutable.

Reply:
Reworked the integration test to not include the full `stdout`, but rather specific elements. Seems files added to `./fixtures` don't get included in the CI. Also the `gas_used` might change in the future.